docs/src/components/navigation/componentPreviewTabs.tsx

```diff
@@ -57,10 +57,19 @@ function getNumber(location: string) {
     }
 }
 
+const tabStyles = {
+    width: '33%',
```

Comment:
Will 33% always work or should we be managing this with flex?

Reply:
I tried with display flex and justify content space evenly, and is better than 33%. We well be covering that point in the DrawerFooter API docs pull-request.